modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/ProteinFeatures_conf.pm

```diff
@@ -47,7 +47,7 @@ sub default_options {
     max_dirs_per_directory  => $self->o('max_files_per_directory'),
 
     # InterPro settings
-    interproscan_version    => '5.46-81.0',
+    interproscan_version    => '5.48-83.0',
```

Comment:
Is is sensible to use some kind of "get whatever version is available by the time you run the pipeline"? 
for instance, with RR, we do:
``
INTERPRO_VERSION=`interproscan.sh -version | egrep version | tr -s ' ' | cut -d ' ' -f 3`
``

Then pass the version to the pipeline, but I guess this could be automated as well. right?

Reply:
It would be handy to be able to tell the pipeline to use whatever version is available on the farm, but you would also need to determine the source db versions automatically - if that turns out to be not too tricky, I can add it to this PR. If not, I think the SOP for RR will need to change, and we'll have to work out how to get it to use the correct conf file without passing in the version; because the config also contains the versions for the different InterPro sources, passing the interproscan_version parameter would break the new version-checking functionality.

Reply:
It definitely need more thinking, this is fine for this PR to stay the same. We'll think again about it at some point (More generally speaking about having something reliable to get the current software versions available in our software environment). Wasn't VersionService not an attempt to do that?

Reply:
Ah, just noticed that this discussion is on the wrong PR - this one is just for the RNA Gene Xref pipeline, the Protein Features one is #541. The interproscan version only appears here because #540 has not been merged yet, once it has, those commits will disappear from this PR.